gravitee-am-cypress/cypress.json

```diff
@@ -0,0 +1,9 @@
+{
+  "baseUrl": "http://localhost/am/ui",
```

Comment:
Is it possible to move the baseUrl into env section?
Will be probably more flexible and allow us to run easily the tests on multiple platform.